src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -36,14 +36,37 @@
   </Node>
 
   <!--
-  Bound node that represents the binding of an "= Value" construct in a declaration.
+  Bound node that represents the binding of an "= Value" construct in a field declaration.
   Appears only in bound trees generated by a SemanticModel.
   -->
-  <Node Name="BoundEqualsValue" Base="BoundNode">
-    <!-- Expression representing the value. -->
-    <Field Name="Value" Type="BoundExpression"/>
+  <Node Name="BoundFieldEqualsValue" Base="BoundEqualsValue">
```

Comment:
I added three derived node types to indicate the kind of initialized symbol. This could be done by instead adding a field of type ISymbol to BoundEqualsValue, should that be preferable.